integration-tests/deploy_R_models.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import absolute_import, print_function
```

Comment:
I think you're right @dubeyabhi07. Leave the tests as integration tests in the same style as those for `deploy_pyspark_models.py`. The python unittests framework is really for single function unit tests that run in a single Python process and don't have side effects.